raiden/storage/migrations/v19_to_v20.py

```diff
@@ -178,6 +216,7 @@ def upgrade_v19_to_v20(
         **kwargs,
 ) -> int:
     if old_version == SOURCE_VERSION:
+        _add_onchain_locksroot_to_channel_new_state_changes(raiden, storage)
```

Comment:
Shouldn't you also add a test for this? That the bug you are fixing is indeed handled? So that a `ContractReceiveChannelNew` did not have `onchain_locksroot` and now it does? Perhaps it's already done since I see your previous test data had that attribute and now you deleted it, but I want to make sure.

Reply:
Done

Reply:
Thank you for adding the test.